README.md

```diff
@@ -1,7 +1,7 @@
 # ampersand-state
 
 <!-- starthide -->
-Part of the [Ampersand.js toolkit](http://ampersandjs.com) for building clientside applications.
+Part of the [Ampersand.js toolkit](http://ampersandjs.com) for building client side applications.
```

Comment:
All the other readmes (i.e. https://github.com/AmpersandJS/ampersand-model) say clientside.